static/app/views/dashboards/widgetCard/widgetCardContextMenu.tsx

```diff
@@ -276,6 +279,16 @@ function WidgetCardContextMenu({
                     {t('Indexed')}
                   </SampledTag>
                 )}
+              {description && (
+                <Tooltip title={description} containerDisplayMode="grid" isHoverable>
+                  <Button
+                    aria-label={t('Widget description')}
+                    borderless
+                    size="xs"
+                    icon={<IconInfo />}
+                  />
+                </Tooltip>
```

Comment:
Last nit, this should probably be `QuestionTooltip`

Reply:
after discussing on slack, we're gunna keep using `Button` for now because it allows tab accessibility